tests/java-optionals/build.gradle.kts

```diff
@@ -0,0 +1,47 @@
+plugins {
+  id("org.jetbrains.kotlin.jvm")
```

Comment:
It might make sense to write these tests in Java to double check ergonomics. In particular, I'm expecting Java users to call `Optional.present(value)` instead of `new Optional.Present(value)` and all these kind of things.

Reply:
```suggestion
  id("java")
```